cfme/tests/infrastructure/test_provisioning_rest.py

```diff
@@ -111,6 +111,43 @@ def test_provision(request, appliance, provider, provision_data):
     assert found_vms, 'VM `{}` not found'.format(vm_name)
 
 
+@pytest.mark.rhv2
+@pytest.mark.uncollectif(lambda provider: not provider.one_of(RHEVMProvider),
+                         reason='These profile names are RHV specific.')
+@pytest.mark.parametrize('vnic_profile', ['no_vnic_profile', 'profile_name(network_name)'])
+def test_provision_vlan(request, appliance, provision_data, vnic_profile, provider):
+    """Tests provision via REST API.
+    Prerequisities:
+        * Have a provider set up with templates suitable for provisioning.
+    Steps:
+        * POST /api/provision_requests (method ``create``) the JSON with provisioning data. The
+            request is returned.
+        * Apart from the usual provisioning settings, set vlan
+          to values <Empty>/profile_name (network_name)
+        * Query the request by its id until the state turns to ``finished`` or ``provisioned``.
+    Metadata:
+        test_flag: rest, provision
+    """
+    vm_name = provision_data['vm_fields']['vm_name']
+    if vnic_profile == 'no_vnic_profile':
+        vnic_profile = '<Empty>'
+    elif vnic_profile == 'profile_name(network_name)':
+        vlan_data = provider.data["provisioning"]["vlan"]
+        vnic_profile = vlan_data + ' (' + vlan_data + ')'
```

Comment:
I'm thinking this might look a little bit more readable: vnic_profile =  '{0} ({0})'.format(vlan_data)

Reply:
Fixed